templates/common/infra/bicep/core/host/container-apps-auth.bicep

```diff
@@ -0,0 +1,42 @@
+metadata description = 'Creates an Azure Container Apps Auth Config using Microsoft Entra as Identity Provider.'
+
+@description('The name of the container apps resource within the current resource group scope')
+param name string
+
+@description('The client ID of the Microsoft Entra application.')
+param clientId string
+
+@description('The name of the Container Apps secret that contains the client secret of the Microsoft Entra application.')
+param clientSecretName string
+
+@description('The OpenID issuer of the Microsoft Entra application.')
```

Comment:
Can you give an example of the issuer.  Looks like the docs have more info.  Maybe we just ask for the tenant id and create the issue for them?

The OpenID Connect Issuer URI that represents the entity which issues access tokens for this application.
When using Azure Active Directory, this value is the URI of the directory tenant, e.g. https://login.microsoftonline.com/v2.0/{tenant-guid}/.
This URI is a case-sensitive identifier for the token issuer.
More information on OpenID Connect Discovery: http://openid.net/specs/openid-connect-discovery-1_0.html

Reply:
The issuer depends on if it is Microsoft Entra ID or Microsoft Entra External ID.  What you have above is for Microsoft Entra ID.  for Microsoft Entra External ID it will look like https://<tenant-domain>.ciamlogin.com/<tenant-id>/v2.0 .This is provided to the user in the Entra ID Portal for their tenant.